package.json

```diff
@@ -29,7 +29,7 @@
     "docs": "node ./scripts/docs/packages.js",
     "bundle": "node ./scripts/docs/bundle.js",
     "lint": "jshint scripts/ packages/ system-test/ test/ && jscs packages/ system-test/ test/",
-    "test": "npm run docs && mocha test/docs.js packages/*/test/*.js",
+    "test": "npm run docs && npm run bundle && mocha test/docs.js packages/*/test/*.js",
```

Comment:
I'll have to learn more about this later. I just assumed it was necessary to run before 'npm test'. If not and its the only thing holding the PR back, can you just change it on my fork/branch?